File: python/1010.py
peca1 = input().split()
valor_peca_1 = (float(peca1[1]) * float(peca1[2]))

peca2 = input().split()
valor_peca_2 = (float(peca2[1]) * float(peca2[2]))

preco_final = valor_peca_1 + valor_peca_2

print(f"VALOR A PAGAR: R$ {preco_final:.2f}")

File: lua/1010.lua
local function split(str)
    local r = {}

    for p in str:gmatch("([^%s]+)") do
        table.insert(r, p)
    end

    return r
end

local p1 = split(io.read())
local p2 = split(io.read())

local total = p1[2] * p1[3] + p2[2] * p2[3]

total = string.format("%.2f", total)

print("VALOR A PAGAR: R$ "..total)
